meinberlin/apps/plans/exports.py

```diff
@@ -19,7 +19,6 @@ class DashboardPlanExportView(
     export_views.BaseExport,
     export_views.AbstractXlsxExportView,
 ):
-
```

Comment:
not sure what removes this line automatically. But restoring it does not work (it considers this as empty commit). Shall I have another look?

Reply:
I think black does that. I don't know why it does do that now and not earlier, but it's fine, that it does. So, leave it!